migen/build/platforms/mercury.py

```diff
@@ -94,7 +97,7 @@
         # Subsignal("oe_n", Pins()),  # If OE wasn't tied to ground on Mercury,
         # this pin would be here.
         IOStandard("LVTTL"), Misc("SLEW=FAST")
-    )
+     )
```

Comment:
why is there a new space here? pep8?

Reply:
Yes, it is technically correct according to pep8, though I must've "fixed" this absentmindedly. Reverting.